tests/holdings_maintenance/support.py

```
@@ -0,0 +1,644 @@
+
+    Attributes:
+        argv: The command line that was run.
+        returncode: The tool's exit code.
+        stdout: What the tool printed -- its logger output and its reports.
+        stderr: Everything else the process wrote: tracebacks, and any warning an
+            imported library chose to emit.
+        output: stdout followed by stderr, for assertions that do not care which
+            stream a message arrived on.
+    """
+
+    def __init__(self, argv, returncode, stdout, stderr):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)